plugin/src/helpers/files.ts

```diff
@@ -27,6 +28,31 @@ const RELOCATABLE_BINARIES = [
   `node.abi${DEFAULT_LAMBDA_ABI}.glibc.node`,
 ]
 
+export const setupDecoupledSourcing = async (baseDir: string) => {
+  // Use experimental decoupled sourcing
+  if (
+    !process.env.GATSBY_CLOUD_DATALAYER ||
+    !process.env.GATSBY_SITE_ID ||
+    !process.env.SITE_AUTH_JWT
+  ) {
```

Comment:
These currently need to be manually set

Reply:
instead of SITE_AUTH_JWT, can we make it `RESOURCE_AUTH_JWT`

Reply:
and instead of `GATSBY_SITE_ID` lets do `CONTENT_CLOUD_ID`

Reply:
`SITE_AUTH_JWT` is for `merlin-synchronizer`, so we'd need to change that module to allow either